server.py

```diff
@@ -59,6 +64,6 @@ def do_POST(self):
             SimpleHTTPRequestHandler.do_POST(self)
 
 if __name__ == '__main__':
-    print "Server started: http://localhost:3000/"
+    print("Server started: http://localhost:3000/")
```

Comment:
I'm aware that adding a comma and another string would turn it into a tuple, however this is not the case.

I'd rather avoid unnecessary imports.